build-logic/jvm/src/main/kotlin/gradlebuild.unittest-and-compile.gradle.kts

```diff
@@ -50,7 +50,7 @@ fun configureCompile() {
         languageVersion.set(JavaLanguageVersion.of(11))
         // Do not force AdoptOpenJDK vendor for M1 Macs
         if (!OperatingSystem.current().toString().contains("aarch64")) {
-            vendor.set(JvmVendorSpec.ADOPTOPENJDK)
+            vendor.set(JvmVendorSpec.ADOPTIUM)
```

Comment:
Do we have an adoptium JDK 11 already installed on CI and is it passed in as well? I suppose we'd also need to change something in our build configuration, right?

What effect does this have on developers? Do I need to install an Adoptium JDK to run the build now if I want to avoid downloading it automatically?

Reply:
> Do we have an adoptium JDK 11 already installed on CI and is it passed in as well? I suppose we'd also need to change something in our build configuration, right?

I'm unsure, CI configuration would need changing for sure.

> What effect does this have on developers? Do I need to install an Adoptium JDK to run the build now if I want to avoid downloading it automatically?

Yes, though given that AdoptOpenJDK now only serves Adoptium builds, this is essentially required for all people who start working on GBT anyways. It only really affects those with prior provisioned JDKs. JDKs from tools such as SDKMAN are likely already Adoptium anyways, as that's the vendor for the latest patch releases.